interface.go

```diff
@@ -97,6 +102,8 @@ func NewInterface(c *InterfaceConfig) (*Interface, error) {
 		udpQueues:          c.udpQueues,
 		tunQueues:          c.tunQueues,
 		version:            c.version,
+		writers:            make([]*udpConn, c.udpQueues),
```

Comment:
We can simplify the config and tun structs to only have `routines` now instead of a distinct `tun` and `udp` queue counts